### ISSUE TITLE
Safari does not support scroll-padding (without a scroll snap container)

### ISSUE THREAD

**Issue by Dantemss** [mdn/browser-compat-data#4497]
Version tested: Safari 12.1 (OS X)

Seems to have been reported to Webkit in 2017 but is still unassigned: https://bugs.webkit.org/show_bug.cgi?id=179379

According to the spec (also linked in the bug report above), scroll-padding should work regardless of scroll-snap-type: https://drafts.csswg.org/css-scroll-snap/#scroll-padding

Simple test page:

```html
<!doctype html>
<html lang=en style="scroll-padding-top: 10vh; overflow: auto;">
  <head>
    <title>scroll-padding test</title>
  </head>
  <body>
    <div style="min-height: 100vh;">
      <a href="#mid">Scroll to mid</a>
    </div>
    <div style="min-height: calc(10vh - 1em);">
      This text is ~10vh - 1em above mid
    </div>
    <div style="min-height: 100vh;">
      <a id="mid" href="#mid">This is mid</a>
    </div>
    <a href="#mid">Scroll to mid</a>
  </body>
</html>
```

The text above mid is visible in Chome and Firefox when clicking any of the anchors, but not in Safari.

**Comment by Dantemss** [comment]
Interestingly enough, moving the `scroll-padding-top` and `overflow` from `<html>` to `<body>` doesn't work on Firefox 68. Still works on Chrome and still doesn't work on Safari.

**Comment by vinyldarkscratch** [comment]
It looks like this may have been fixed by #5803, so I'm going to close this issue.  Happy to reopen this if I'm mistaken!
